easybuild/easyblocks/p/perl.py

```diff
@@ -64,6 +65,9 @@ def configure_step(self):
             '-Dinstallscript=%(installdir)s/bin',
             '-Dscriptdir=%(installdir)s/bin',
             '-Dscriptdirexp=%(installdir)s/bin',
+            # guarantee that the install directory has the form lib/perlX/
+            # see https://github.com/easybuilders/easybuild-easyblocks/issues/1700
+            "-Dinstallstyle = 'lib/perl%s'" % majver,
```

Comment:
@ocaisa let's drop the spaces around `=` here, to be in sync with the other configure options

Reply:
This may cause this problem I'm seeing when testing rebuild of `Perl` using this updated easyblock...

```
Sanity check failed: no (non-empty) directory found at 'lib/perl5/5.28.1'
```